index.bs

```diff
@@ -12471,112 +12456,27 @@ The characteristics of a namespace object are described in [[#namespace-object]]
-
-<div algorithm="to create a DOMException">
-
-    When the DOMException function is called with arguments |message| and |name|,
-    the following steps are taken:
-
-    1.  Let |F| be the active function object.
-    1.  If NewTarget is <emu-val>undefined</emu-val>, let |newTarget| be |F|, else let |newTarget| be NewTarget.
-    1.  Let |super| be |F|.\[[GetPrototypeOf]]().
-    1.  [=ReturnIfAbrupt=](|super|).
-    1.  If [=IsConstructor=](|super|) is <emu-val>false</emu-val>, throw a <emu-val>TypeError</emu-val> exception.
-    1.  Let |O| be [=Construct=](|super|, «|message|», |newTarget|).
-    1.  If |name| is not <emu-val>undefined</emu-val>, then
-        1.  Let |name| be [=ToString=](|name|).
-        1.  Let |status| be [=DefinePropertyOrThrow=](|O|, <code>"name"</code>, PropertyDescriptor{\[[Value]]: |name|, \[[Writable]]: <emu-val>true</emu-val>, \[[Enumerable]]: <emu-val>false</emu-val>, \[[Configurable]]: <emu-val>true</emu-val>}).
-        1.  [=ReturnIfAbrupt=](|status|).
-        1.  Let |code| be the legacy code indicated in the [=error names table=] for error name |name|, or <emu-val>0</emu-val> if there is none.
-        1.  Let |status| be [=DefinePropertyOrThrow=](|O|, <code>"code"</code>, PropertyDescriptor{\[[Value]]: |code|, \[[Writable]]: <emu-val>true</emu-val>, \[[Enumerable]]: <emu-val>false</emu-val>, \[[Configurable]]: <emu-val>true</emu-val>}).
-        1.  [=ReturnIfAbrupt=](|status|).
-    1.  Return |O|.
-</div>
-
+<h4 id="es-DOMException-specialness" oldids="es-DOMException-constructor-object, es-DOMException-prototype-object">{{DOMException}} custom bindings</h4>
 
-<h4 id="es-DOMException-prototype-object">DOMException prototype object</h4>
-
-The DOMException prototype object must
-have an internal \[[Prototype]] property whose value is [=%ErrorPrototype%=].
-
-The [=class string=] of the
-[=DOMException prototype object=]
-is “DOMExceptionPrototype”.
-
-There must be a property named “constructor”
-on the DOMException prototype object with attributes
-{ \[[Writable]]: <emu-val>true</emu-val>, \[[Enumerable]]: <emu-val>false</emu-val>, \[[Configurable]]: <emu-val>true</emu-val> }
-and whose value is the [=DOMException constructor object=].
+In the ECMAScript binding, the {{DOMException}} type has some additional requirements:
 
-For every legacy code listed in the <a href="#dfn-error-names-table">error names table</a>,
-there must be a property on the DOMException prototype object
-whose name and value are as indicated in the table.  The property has
-attributes { \[[Writable]]: <emu-val>false</emu-val>, \[[Enumerable]]: <emu-val>true</emu-val>, \[[Configurable]]: <emu-val>false</emu-val> }.
+*   Unlike normal [=interface types=], the [=interface prototype object=] for {{DOMException}} must
+    have as its \[[Prototype]] the intrinsic object [=%ErrorPrototype%=].
 
+*   If an implementation gives native <emu-val>Error</emu-val> objects special powers or
```

Comment:
FWIW, I've sent an [email to v8-dev](https://groups.google.com/forum/#!topic/v8-dev/nQVOwXFqVnE) to discuss what we'd need to do in Chromium and there are some questions over how this would interact with https://github.com/tc39/proposal-error-stacks.